sphinx/templates/latex/latex.tex_t

```diff
@@ -48,7 +56,9 @@
 \date{<%= date %>}
 \release{<%= release %>}
 \author{<%= author %>}
-\newcommand{\sphinxlogo}{<%= logo %>}
+<%- if logofilename %>
+\newcommand{\sphinxlogo}{\sphinxincludegraphics{<%= logofilename %>}\par}
+<%- endif %>
```

Comment:
```
<%- if logofilename %>
\newcommand{\sphinxlogo}{\sphinxincludegraphics{<%= logofilename %>}\par}
<%- else %>
\newcommand{\sphinxlogo}{\vbox{}}
<%- endif %>
```

Reply:
oh, thanks!

Reply:
@tk0miya it is also needed to remove mention of ``'logo'`` from this

```
* Keys that are set by other options and therefore should not be overridden
     are:

     ``'docclass'``
     ``'classoptions'``
     ``'title'``
     ``'date'``
     ``'release'``
     ``'author'``
     ``'logo'``
     ``'makeindex'``
```

at lines about L2319-2329 in https://github.com/sphinx-doc/sphinx/blob/master/doc/usage/configuration.rst, and also ``'date'``

Reply:
done!